libipl/p10/ipl0.C

```diff
@@ -153,6 +153,15 @@ static int update_hwas_state_callback(struct pdbg_target* target, void *priv)
 
 	} else if (ipl_type() == IPL_TYPE_NORMAL) {
 
+		if (strcmp(pdbg_target_class_name(target), "proc") == 0) {
+			if (ipl_is_master_proc(target)) {
+				ipl_log(IPL_INFO,
+					"Primary processor is guarded "
+					"so, skipping to apply");
+				return 3;
```

Comment:
3? what does it indicate enums should have been good, but will not bother in the handling code you are checking for 1 and 2 and not 3
http://fstone07p1.aus.stglabs.ibm.com:8080/source/xref/phal/ipl/libipl/p10/ipl0.C?r=4102a541#250

Reply:
0 - Target not found in the device 
> 0 - Application can use based on their needs and already they used 2 for the error case so, using 3.

Reply:
Pushed separate commit to use a macro instead of numeric for better readability.